src/gs-update-monitor.c

```diff
@@ -68,6 +69,20 @@ typedef struct {
 	gulong invalidate_handler_id;
 } UpdateScheduleHelper;
 
+static void
+scheduled_entry_removed_cb (GObject *source_object,
+			    GAsyncResult *result,
+			    GAsyncResult **out_result)
+{
+	MwscScheduleEntry *entry = (MwscScheduleEntry *) source_object;
+	g_autoptr(GError) local_error = NULL;
+
+        if(!mwsc_schedule_entry_remove_finish (entry, result, &local_error))
```

Comment:
Nitpick: Missing space before `(`

Reply:
And there was spaces instead of tabs too...